### PR TITLE
Update read_activations.py

### PR DESCRIPTION
**Pull request by zhhongzhi**
Set the Learning phase to 0 (original if 1) for test time.
In Keras 2.0,
'The learning phase flag is a bool tensor (0 = test, 1 = train)'.

### CODE REVIEW

**Comment by philipperemy** [comment]
@zhhongzhi thanks a lot for this PR!

I'll have a look later on :)

**Comment by philipperemy** [comment]
Refer to #2

**Comment by qihongl** [comment]
Hi, @zhhongzhi @philipperemy 

just want to understand what's going on here (code also copied below): https://github.com/philipperemy/keras-visualize-activations/blob/master/read_activations.py#L27

```
# Learning phase. 0 = Test mode (no dropout or batch normalization)
# layer_outputs = [func([model_inputs, 0.])[0] for func in funcs]
layer_outputs = [func(list_inputs)[0] for func in funcs]
```

Is the current code fetching the "test mode" activity? It doesn't seem to be the case since I'm still getting the dropout layer... And what is the commented line doing? 


Thank you very much in advance!

**Comment by philipperemy** [comment]
Keras has a learning_phase tensor used in many layers such as Dropout and BatchNorm.

When you use .predict(), you don't want it to be random. Dropout is a random layer.

So Keras sets learning_phase to 0 to ensure that the Dropout will become the Identity layer during inference. So you will still see Dropout but should behave like an Identity function.

**Comment by qihongl** [comment]
I see! Thanks!

**Comment by philipperemy** [comment]
@qihongl welcome :)
